### DISCUSSION TITLE
ReactPy - Future after stability

### DISCUSSION THREAD

**Discussion by adk7712** [idom-team/idom#1067] [General]
Will the project aim to create additional features unavailable in native react, once it is stable?

**Comment by Archmonger** [comment]
In terms of features in the `reactive-python/reactpy` repo, it is unlikely we will add features beyond ReactJS capabilities unless forced to due to technological limitations. You can look at our issue tracker to see what features we plan to develop for this repo.

However, we do plan to create other Python packages that extend the core feature set. For example, we currently have `reactive-python/reactpy-router` which adds SPA capabilities.

**Reply by adk7712** [reply]
wow! looking forward to it
